libs/sdk-ui-filters/src/AttributeFilter@next/Components/Addons/AttributeDisplayFormSelect/AttributeFilterDisplayFormSelect.tsx

```diff
@@ -0,0 +1,36 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { ObjRef } from "@gooddata/sdk-model";
+import { IAlignPoint } from "@gooddata/sdk-ui-kit";
+import { AttributeDisplayFormDropdown } from "./AttributeDisplayFormDropdown";
+import { useAttributeFilterContext } from "../../../Context/AttributeFilterContext";
+
+/**
+ * @internal
+ */
+export interface IAttributeFilterDisplayFormSelectProps {
+    onSelect: (displayForm: ObjRef) => void;
+    alignPoints?: IAlignPoint[];
+}
+
+/**
+ * @internal
+ */
+export const AttributeFilterDisplayFormSelect: React.FC<IAttributeFilterDisplayFormSelectProps> = (props) => {
+    const { onSelect, alignPoints } = props;
+
+    const { displayForms, currentDisplayFormRef } = useAttributeFilterContext();
+
+    if (displayForms.length <= 1) {
```

Comment:
This condition seems like a something that should be in AD :/

Reply:
Done